Driver/Driver/Helper/Animator.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  Animator.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 22.02.2023.
+//
+
+import UIKit
+
+public class Animator {
+    public static let sharedInstance = Animator()
```

Comment:
**sharedInstance** is an old way definition. Please, use just shared without public.
 ` static let shared: Animator = Animator()
`